Spokestack/SpeechEventListener.swift

```diff
@@ -14,33 +14,37 @@ import Foundation
 @objc public protocol SpeechEventListener: AnyObject {
     
     /// The speech pipeline has been initialized.
-    func didInit() -> Void
+    @objc func didInit() -> Void
     
     /// The speech pipeline has been started.
-    func didStart() -> Void
+    @objc func didStart() -> Void
     
     /// The speech pipeline has been stopped.
-    func didStop() -> Void
+    @objc func didStop() -> Void
     
     /// The pipeline activate event. Occurs upon activation of speech recognition.  The pipeline remains active until the user stops talking or the activation timeout is reached.
     /// - SeeAlso:  wakeActiveMin
-    func didActivate() -> Void
+    @objc func didActivate() -> Void
     
     /// The pipeline deactivate event. Occurs upon deactivation of speech recognition.  The pipeline remains inactive until activated again by either explicit activation or wakeword activation.
-    func didDeactivate() -> Void
+    @objc func didDeactivate() -> Void
     
     /// The pipeline recognized and transcribed speech.
     /// - Parameter result: The speech pipeline context, which contains the result.
-    func didRecognize(_ result: SpeechContext) -> Void
+    @objc func didRecognize(_ result: SpeechContext) -> Void
+    
+    /// The  pipeline recognized and transcripted an incomplete portion of the utterance.
```

Comment:
```suggestion
    /// The pipeline recognized and transcribed a portion of an incomplete utterance.
```
Based on that description, isn't it more `didRecognizePartial` than `didPartiallyRecognize`?

Reply:
I can go with that